fvm/bootstrap.go

```diff
@@ -181,6 +181,7 @@ func (b *BootstrapProcedure) Run(vm *VirtualMachine, ctx Context, sth *state.Sta
 
 	service := b.createServiceAccount(b.serviceAccountPublicKey)
 
+	b.deployContractAuditVouchers(service)
```

Comment:
does the order of deployment matter? shouldn't it be after deploying fungible token and flow tokens contracts?

Reply:
The order changes the hardcoded root hash. The `FlowContractAudits` contract doesn't import any other contracts and is used for deployments so I chose to deploy in this order.